packages/react-static/src/static/webpack/rules/cssLoader.js

```diff
@@ -17,13 +17,14 @@ function initCSSLoader() {
         // Necessary for external CSS imports to work
         // https://github.com/facebookincubator/create-react-app/issues/2677
         sourceMap: true,
-        ident: 'postcss',
-        plugins: () => [
```

Comment:
This is a change with the postcss-loader 4.0 configuration API - this is able to be an array and the key is part of `postcssOptions`

https://github.com/webpack-contrib/postcss-loader/blob/435224d134f6193d42859988e047b6dbe8dcdbce/CHANGELOG.md#-breaking-changes

>>> PostCSS (plugins/syntax/parser/stringifier) options was moved to the postcssOptions option, please look at [docs](https://github.com/webpack-contrib/postcss-loader#postcssoptions)

Reply:
This also technically breaks updates to the `node.api.js` if the user attempts to find and append to the plugins as there is another object to dig through

eg.
```
  const plugins = postcssLoader.options.plugins;

  Object.assign(postcssLoader.options, { 
    plugins: [...new_plugins, plugins]
  }
```

must be changed to
```
  const plugins = postcssLoader.options.postcssOptions.plugins;
  
  Object.assign(postcssLoader.options.postcssOptions, {
    plugins: [...new_plugins, plugins]
  } 
```

I'm not entirely sure what the community conventions around "breaking changes" referring to webpack config is - but that might push this to a minor version upgrade instead of a patch :shrug:

Reply:
If it's a breaking change, it _should_ be a major bump but 🤷🏽‍♂️

Reply:
yeah, :point_up: is the equivalent of monkey patching a runtime method in Ruby for webpack plugins imo but I'm not sure if anyone does that

Reply:
Eh. This PR is linked in the CHANGELOG, including this conversation 😋